libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list-item.component.spec.ts

```diff
@@ -31,4 +32,14 @@ describe(WattNavListItemComponent.name, () => {
 
     expect(component.link).toBeNull();
   });
+
+  it('`target` input property is `_self` by default', async () => {
+    const view = await render(WattNavListItemComponent, {
+      imports: [WattNavListItemScam],
+    });
+
+    const component = view.fixture.componentInstance;
+
+    expect(component.target).toBe('_self');
+  });
```

Comment:
suggestion: Nice isolated unit tests but consider making it a component test by verifying through the DOM instead and include the `WattNavListComponent` to make it an integration test.